streams/integration-tests/src/test/java/org/apache/kafka/streams/integration/SmokeTestDriverIntegrationTest.java

```diff
@@ -120,6 +120,8 @@ SmokeTestDriver.VerificationResult result() {
         "true, false, true",
         "false, false, false",
         "true, false, false",
+        "true, true, true",
```

Comment:
Are the proccessing thread test combinations stable now?

Reply:
Yes, this was solved by the unification of the queues. Before I missed to add the event handling method to the `runOnce` for processing threads.